
From: Andrew James Wade <ajwade@cpe00095b3131a0-cm0011ae8cd564.cpe.net.cable.rogers.com>
Subject: Fix build dependencies for reiser4

- ZLIB_INFLATE is not visible in menuconfig.  Reiser4 should probably
  just select it like the other filesystems do.

- Reiser4 also depends on ZLIB_DEFLATE.

Signed-off-by: Andrew Wade <ajwade@alumni.uwaterloo.ca>
Cc: <reiserfs-dev@namesys.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Acked-by: Hannes Reinecke <hare@suse.de>

---

 fs/reiser4/Kconfig |    4 +++-
 1 files changed, 3 insertions(+), 1 deletion(-)

diff -puN fs/reiser4/Kconfig~reiser4-fix-dependencies fs/reiser4/Kconfig
--- 25/fs/reiser4/Kconfig~reiser4-fix-dependencies	2005-06-24 01:53:14.000000000 -0700
+++ 25-akpm/fs/reiser4/Kconfig	2005-06-24 01:53:14.000000000 -0700
@@ -1,6 +1,8 @@
 config REISER4_FS
 	tristate "Reiser4 (EXPERIMENTAL)"
-	depends on EXPERIMENTAL && !4KSTACKS && ZLIB_INFLATE
+	depends on EXPERIMENTAL && !4KSTACKS
+	select ZLIB_INFLATE
+	select ZLIB_DEFLATE
 	help
 	  Reiser4 is a filesystem that performs all filesystem operations
 	  as atomic transactions, which means that it either performs a
_
